api/controllers/InvitationController.js

```diff
@@ -125,5 +125,26 @@ module.exports = {
         })
       })
     .then(results => res.ok({results}))
+  },
+
+  reInviteAll: function (req, res) {
+    const communityId = res.locals.community.id
+    return Invitation.where({community_id: communityId, used_by_id: null})
```

Comment:
if this is onerous (it is a pretty ugly query), an alternative would be to create an instance method on User that searches for an Invitation with matching email and marks it used, and then calling it during User.create